### DISCUSSION TITLE
Control crazyflie from a swam individually?

### DISCUSSION THREAD

**Discussion by ghostSAS** [USC-ACTLab/crazyswarm#729] [Q&A]
Hi all,

I am currently working on a project that needs to control each crazyflie from a swarm individually. For example, I need cf1 to continue to track a given trajectory, while cf2 starts to land. As my current work is built upon someone else's that was written in crazyswarm, I think it would be better to stick with crazyswarm for faster implementation. 

If there is no natural way to do that, I believe that I have to assign every crazyflie in a thread.

Any hint and suggestion would be appreciated!

**Comment by whoenig** [comment]
Perhaps https://github.com/USC-ACTLab/crazyswarm/blob/master/ros_ws/src/crazyswarm/scripts/backgroundComputation.py helps as an example.

**Reply by ghostSAS** [reply]
Hi Wolfgang, 

thanks for your answer, and sorry for the late response. I managed to "virtually" control them individually by asking them to follow differently corresponding waypoints via cmdPosition(). 

I hope it also gives some inspiration to someone who wants to achieve similar tasks.
